Documentation/network/egress-gateway/egress-gateway.rst

```diff
@@ -303,6 +303,44 @@ setting the ``--devices`` agent option accordingly.
    gateway node's network configuration (for example if an IP address is added or deleted). You can force a fresh selection by re-applying the
    Egress Gateway policy.
 
+Selecting multiple gateway nodes
+--------------------------------
+
+It's possible to select multiple gateway nodes in the same policy. In this case, the gateway nodes
+can be configured using the ``egressGateways`` list field. Entries on this list have the exact same
+configuration options as the ``egressGateway`` field:
+
+.. code-block:: yaml
+
+  egressGateways:
+  - nodeSelector:
+      matchLabels:
+        testLabel: testVal1
+  - nodeSelector:
+      matchLabels:
+        testLabel: testVal2
+
+.. note::
+
+    The same restrictions as with the ``egressGateway`` field apply to each item of the
+    ``egressGateways`` list.
+
+.. note::
+
+    When using multiple gateways the source endpoints matched by the policy will still egress traffic
+    through a single gateway, not all of them. The endpoints will be assigned to a gateway based on
+    its CiliumEndpoint's UID. Hence, an endpoint should use the same gateway during its lifetime
+    as long as the gateway nodes matched by the ``nodeSelector`` fields don't change. If a
+    ``nodeSelector`` field is added, removed, or modified, or if a node matching one of the
+    ``nodeSelector`` fields is added or removed, the list of gateways will change and the endpoints
+    will be reassigned.
+
+.. warning::
+
+    As with single-gateway policies, changing the gateway node will cause existing egress conections
+    to blackhole traffic. There is already a
```

Comment:
> There is already Github discussion to mitigate this issue.

Can we reword this a bit, maybe just say something like:

The following [Github discussion](https://github.com/cilium/cilium/discussions/39245) tracks this issue. 

As a tangent, I've never seen a github discussion referenced in the docs, grepping through the docs pages we commonly just reference an issue. Wonder if we should turn this back to an issue to reference here (cc @joestringer thoughts?)